int32 publish_count
float32 grav_x
bool is_level
